service/responses.tsp

```diff
@@ -0,0 +1,152 @@
+import "@typespec/http";
+import "@typespec/openapi";
+
+using Http;
+
+@tag("Responses")
+namespace CoreSystem.Responses {
+  @doc("The available answer types")
+  enum AnswerType {
+    shortText: "short_text",
+    longText: "long_text",
+    singleChoice: "single_choice",
+    multipleChoice: "multiple_choice",
+    date: "date",
+  }
+
+  @doc("A form response submission")
+  model Response {
+    @doc("The response's unique identifier.")
+    id: uuid;
+
+    @doc("The form that this response belongs to.")
+    formId: uuid;
+
+    @doc("The user who submitted this response.")
+    submittedBy: uuid;
+
+    @doc("The creation timestamp of the response.")
+    createdAt: utcDateTime;
+
+    @doc("The last updated timestamp of the response.")
+    updatedAt: utcDateTime;
+  }
+
+  @doc("An individual answer")
+  model Answer {
+    @doc("The answer's unique identifier.")
+    id: uuid;
+
+    @doc("The response this answer belongs to.")
+    responseId: uuid;
+
+    @doc("The question this answer is for.")
+    questionId: uuid;
+
+    @doc("The type of answer, determines how to parse answerValue.")
+    answerType: AnswerType;
```

Comment:
Consider renaming `answerType` to just `type` for brevity and consistency, while keeping the data type as `AnswerType`. Since the context is already within Answer, the prefix feels redundant.